src/Elliptic/Actions/InitializeBackgroundFields.hpp

```diff
@@ -68,8 +68,13 @@ struct InitializeBackgroundFields {
     const auto& background = db::get<BackgroundTag>(box);
```

Comment:
could you elaborate in the commit message (briefly) how this fixes anything? I have no idea what to make of this change

Reply:
Some background fields need numeric derivatives. The `Binary` analytic data class and the others already support this, but they need the mesh and the inverse Jacobian. The change in this commit makes the action pass those along. I added some comments to the docs and to the commit message.